openfisca_france_inheritance/parameters/droits_mutation_titre_gratuit/bareme/ligne_directe.yaml

```
@@ -0,0 +1,348 @@
+      value: 0.15
+    2013-01-01:
+      value: 0.15
+    2014-12-30:
+      value: 0.15
+- threshold:
+    1979-07-01:
+      value: 100000
+    1984-01-01:
+      value: 100000
+    2002-01-01:
+      value: 15000
+    2008-01-01:
+      value: 15000
+    2009-01-01:
+      value: 15195
+    2010-01-01:
+      value: 15636
+    2011-01-01:
+      value: 15697
+    2012-01-01:
+      value: 15932
+    2013-01-01:
+      value: 15932
+    2014-12-30:
+      value: 15932
+  rate:
+    1979-07-01:
+      value: 0.2
+    1984-01-01:
+      value: 0.2
+    2002-01-01:
+      value: 0.2
+    2008-01-01:
+      value: 0.2
+    2009-01-01:
+      value: 0.2
+    2010-01-01:
+      value: 0.2
+    2011-01-01:
+      value: 0.2
+    2012-01-01:
+      value: 0.2
+    2013-01-01:
+      value: 0.2
+    2014-12-30:
+      value: 0.2
+- threshold:
+    1984-01-01:
+      value: 3400000
+    2002-01-01:
+      value: 520000
+    2008-01-01:
+      value: 520000
+    2009-01-01:
+      value: 526760
+    2010-01-01:
+      value: 542043
+    2011-01-01:
+      value: 544173
+    2012-01-01:
+      value: 552324
+    2013-01-01:
+      value: 552324
+    2014-12-30:
+      value: 552324
+  rate:
+    1984-01-01:
+      value: 0.3
+    2002-01-01:
+      value: 0.3
+    2008-01-01:
+      value: 0.3
+    2009-01-01:
+      value: 0.3
+    2010-01-01:
+      value: 0.3
+    2011-01-01:
+      value: 0.3
+    2012-01-01:
+      value: 0.3
+    2013-01-01:
+      value: 0.3
+    2014-12-30:
+      value: 0.3
+- threshold:
+    1984-01-01:
+      value: 5600000
+    2002-01-01:
+      value: 850000
+    2008-01-01:
+      value: 850000
+    2009-01-01:
+      value: 861050
+    2010-01-01:
+      value: 886032
+    2011-01-01:
+      value: 889514
+    2012-01-01:
+      value: 902838
+    2013-01-01:
+      value: 902838
+    2014-12-30:
+      value: 902838
+  rate:
+    1984-01-01:
+      value: 0.35
+    2002-01-01:
+      value: 0.35
+    2008-01-01:
+      value: 0.35
+    2009-01-01:
+      value: 0.35
+    2010-01-01:
+      value: 0.35
+    2011-01-01:
+      value: 0.35
+    2012-01-01:
+      value: 0.4
+    2013-01-01:
+      value: 0.4
+    2014-12-30:
+      value: 0.4
+- threshold:
+    1984-01-01:
+      value: 11200000
+    2002-01-01:
+      value: 1700000
+    2008-01-01:
+      value: 1700000
+    2009-01-01:
+      value: 1722100
+    2010-01-01:
+      value: 1772064
+    2011-01-01:
+      value: 1779029
+    2012-01-01:
+      value: 1805677
+    2013-01-01:
+      value: 1805677
+    2014-12-30:
+      value: 1805677
+  rate:
+    1984-01-01:
+      value: 0.4
+    2002-01-01:
+      value: 0.4
+    2008-01-01:
+      value: 0.4
+    2009-01-01:
+      value: 0.4
+    2010-01-01:
+      value: 0.4
+    2011-01-01:
+      value: 0.4
+    2012-01-01:
+      value: 0.45
+    2013-01-01:
+      value: 0.45
+    2014-12-30:
+      value: 0.45
+metadata:
+  short_label: Descendants en ligne directe
+  last_value_still_valid_on: "2023-04-01"
```

```suggestion
  last_value_still_valid_on: "2025-06-12"
```

Vérifié sur https://www.kwiper.fr/blog/dmtg/